ui/tests/acceptance/vault-reporting/index-test.js

```diff
@@ -0,0 +1,216 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { visit, currentURL, waitFor } from '@ember/test-helpers';
+import { login } from 'vault/tests/helpers/auth/auth-helpers';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { mockedResponseWithData, mockedEmptyResponse } from 'vault/tests/helpers/vault-usage/mocks';
+
+module('Acceptance | enterprise vault-reporting', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function () {
+    await login();
+  });
+
+  test('it visits the usage reporting dashboard and renders the header', async function (assert) {
+    await visit('/vault/usage-reporting');
+    assert.strictEqual(currentURL(), '/vault/usage-reporting', 'navigates to usage reporting dashboard');
+    assert.dom('.hds-page-header').includesText('Vault Usage', 'renders the "Vault Usage" header');
+  });
+
+  test('it renders the counters dashboard block with all expected counters', async function (assert) {
+    this.server.get('http://localhost:7357/v1/sys/utilization-report', () => mockedResponseWithData);
+    await visit('/vault/usage-reporting');
+    await waitFor('[data-test-vault-reporting-dashboard-counters]');
+    assert
+      .dom('[data-test-vault-reporting-dashboard-counters]')
+      .exists('renders the counters dashboard block');
+
+    const expectedCounters = ['Child namespaces', 'KV secrets', 'Secrets sync', 'PKI roles'];
+
+    expectedCounters.forEach((counterLabel) => {
+      assert
+        .dom(`[data-test-vault-reporting-counter="${counterLabel}"]`)
+        .exists(`counter "${counterLabel}" is rendered`);
+    });
+
+    assert.dom('[data-test-vault-reporting-counter="Child namespaces"]').includesText('1');
+    assert.dom('[data-test-vault-reporting-counter="KV secrets"]').includesText('100');
+  });
+
+  test('dashboard card: Secret engines', async function (assert) {
+    this.server.get('http://localhost:7357/v1/sys/utilization-report', () => mockedResponseWithData);
+    await visit('/vault/usage-reporting');
+    await waitFor('[data-test-vault-reporting-dashboard-secret-engines]');
+
+    assert.dom('[data-test-vault-reporting-dashboard-secret-engines]').exists('renders Secret engines card');
```

Comment:
Not necessary to update, but mostly an FYI. We typically use `const SELECTORS` var at either the top of the test file, or in a separate helper file if they'll be reused elsewhere, to streamline future test selector updates. Here's an [example](https://github.com/hashicorp/vault/blob/82eda875ddb9852d6c52bf80e511f1bd0e81e718/ui/tests/helpers/components/dashboard/dashboard-selectors.js#L7) of a dynamic selector so that the test file just references `SELECTORS.card('secret-engines'`).